### PR TITLE
Increase grpc connection timeout

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
fix TestReachability flake

I have seen it many times, here is one 
https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/26058/integ-security-k8s-tests_istio/17637

```
2020-08-03T06:24:20.683349Z	info	tf	  [ 0]                        vm-v1-67bb7d5cf5-6j994         Running (Ready)
2020-08-03T06:24:20.683758Z	info	tf	  [ 0]                          a-v1-d476f5469-sk9q8         Running (Ready)
    TestReachability: context.go:111: initialize instances: 1 error occurred:
        	* initialize TestReachability/[*kube.instance-2]: grpc client: context deadline exceeded
        
        
2020-08-03T06:24:21.054546Z	info	tf	=== DONE (failed):  Test: 'security[TestReachability] (21.119566006s)' ===
2020-08-03T06:24:21.054571Z	error	tf	=== Dumping Namespace reachability-22-85702 State...
--- FAIL: TestReachability (23.13s)
```

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: pkg/test/echo/common/util.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	ConnectionTimeout     = time.Second * 2
+	ConnectionTimeout     = 5 * time.Second
```
Comment (howardjohn) [inline pkg/test/echo/common/util.go:25]: Please don't change this, it impacts multiple places. I am ok with doing this change to just the single call site and *temporarily* until we resolve these DNS/wasm issues, but we need to create a release blocker issue to revert this change. It is a clear bug that we cannot send traffic for a full 2 seconds, do not want to just hide it

Comment (hzxuzhonghu) [reply pkg/test/echo/common/util.go:25]: Ah, I thought it is the environment issue.

Comment (howardjohn) [reply pkg/test/echo/common/util.go:25]: It does still seem to he failing so I'm not sure why, it's a clear regression. I am fine with changing this if you change it just at the init call site but this variable is used in multiple places and we only want to change the init

Comment (hzxuzhonghu) [reply pkg/test/echo/common/util.go:25]: The case here is: 
 grpc connect to a pod via kube forwarder. No dns resolution at all.

maybe i should change this connection timeout to `pkg/test/echo/client/client.go`

**Comment by howardjohn** [comment]
Can you create an issue and assign to me for this?
